config_fedramp_conformance/variables.tf

```diff
@@ -0,0 +1,155 @@
+variable "access_keys_rotated_max_access_key_age" {
+  description = "Maximum age of access keys in days"
+  type        = number
+  default     = 90
+}
+
+variable "acm_certificate_expiration_check_days_to_expiration" {
+  description = "Number of days before expiration of ACM certificate"
+  type        = number
+  default     = 90
+}
+
+variable "cw_loggroup_retention_period_check_min_retention_time" {
+  description = "Number of days to retain cw logs"
+  type        = number
+  default     = 90
+}
+
+variable "ec2_volume_inuse_check_delete_on_termination" {
+  description = "Checks for ec2 volume delete on termination"
+  type        = string
+  default     = "TRUE"
+}
+
+variable "guard_duty_non_archived_findings_days_low_sev" {
+  description = "Number of days for non archived low sev findings in GuardDuty"
+  type        = string
+  default     = "180"
+}
+
+variable "guard_duty_non_archived_findings_days_medium_sev" {
+  description = "Number of days for non archived medium sev findings in GuardDuty"
+  type        = string
+  default     = "90"
+}
+
+variable "guard_duty_non_archived_findings_days_high_sev" {
+  description = "Number of days for non archived high sev findings in GuardDuty"
+  type        = string
+  default     = "30"
+}
+
+variable "iam_password_policy_max_password_age" {
+  description = "IAM password policy max password age"
+  type        = string
+  default     = "90"
+}
+
+variable "iam_password_policy_minimum_password_length" {
+  description = "IAM password policy minimum password length"
+  type        = string
+  default     = "14"
+}
+
+variable "iam_password_policy_password_reuse_prevention" {
+  description = "IAM password policy password reuse prevention"
+  type        = string
+  default     = "24"
+}
+
+variable "iam_password_policy_require_lowercase_characters" {
+  description = "IAM password policy require lowercase characters"
+  type        = string
+  default     = "TRUE"
+}
+
+variable "iam_password_policy_require_uppercase_characters" {
+  description = "IAM password policy require uppercase characters"
+  type        = string
+  default     = "TRUE"
+}
+
+variable "iam_password_policy_require_numbers" {
+  description = "IAM password policy require numbers"
+  type        = string
+  default     = "TRUE"
+}
+
+variable "iam_password_policy_require_symbols" {
+  description = "IAM password policy require symbols"
+  type        = string
+  default     = "TRUE"
+}
+
+variable "iam_user_unused_credentials_check_max_credential_usage_age" {
+  description = "IAM user credentials unused max age"
+  type        = string
+  default     = "90"
+}
+
+variable "restricted_incoming_traffic_blocked_port1" {
```

Comment:
Suggestion - Let's fold _port1 through _port5 into one map:

~~~
variable "restricted_incoming_traffic_blocked_ports" {
  description = "Map of name: incoming port number to block"
  type        = map(string)
  default = {
    ftpdata  = "20",
    ftp      = "21",
    mysql    = "3306",
    msrdp    = "3389",
    mysqlalt = "4333"
  }
}
~~~

Reply:
These are AWS managed rules the number of parameters and the parameter names are fixed.

Reply:
Someone give the coder behind having a static set of rules named "port1" through "port5" a medal!